dev/RELEASE_PROCESS.rst

```diff
@@ -3,25 +3,65 @@ Release Process
+   passes. *And make sure it is testing the wheels created in step 2!* This
+   will use the autoscaler to start a bunch of machines and run some tests.
+   Any new stress tests should be added to this script so that they will be run
    automatically for future release testing.
 
 3. **Libraries:** Make sure that the libraries (e.g., RLlib, Tune, SGD) are in a
    releasable state. TODO(rkn): These libraries should be tested automatically
    by the script above, but they aren't yet.
 
-4. **Increment the Python version:** Create a PR that increments the Python
-   package version. See `this example`_.
-
-5. **Create a GitHub release:** Create a GitHub release through the `GitHub
+5. **Resolve release-blockers:** Should any release blocking issues arise,
+   there are two ways these issues are resolved: A PR to patch the issue or a
+   revert commit that removes the breaking change from the release. In the case
+   of a PR, that PR should be created against master. Once it is merged, the
+   release master should ``git cherry-pick`` the commit to the release branch.
+   If the decision is to revert a commit that caused the release blocker, the
+   release master should ``git revert`` the commit to be reverted on the
+   release branch. Push these changes directly to the release branch.
+
+6. **Download all the wheels:** Now the release is ready to begin final
+   testing. The wheels are automatically uploaded to S3, even on the release
+   branch. The wheels can be found at the follow URLS Where ``<hash>`` is 
+   the SHA from the release branch most recent commit and ``<version>`` 
+   is the version incremented in Part 1:
+
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp27-cp27mu-manylinux1_x86_64.whl``
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp35-cp35m-manylinux1_x86_64.whl``
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp36-cp36m-manylinux1_x86_64.whl``
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp37-cp37m-manylinux1_x86_64.whl``
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp27-cp27m-macosx_10_6_intel.whl``
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp35-cp35m-macosx_10_6_intel.whl``
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp36-cp36m-macosx_10_6_intel.whl``
+* ``https://s3-us-west-2.amazonaws.com/ray-wheels/<hash>/ray-<version>-cp37-cp37m-macosx_10_6_intel.whl``
+
+7. **Final Testing:** Send a link to the wheels to the other contributors and
+   core members of the Ray project. Make sure the wheels are tested on Ubuntu,
+   Mac OSX 10.12, and Mac OSX 10.13+. This testing should verify that the
+   wheels are correct and that all release blockers have been resolved. Should
+   a new release blocker be found, repeat steps 5-7.
+
+8. **Create a GitHub release:** Create a GitHub release through the `GitHub
```

Comment:
Let's move this to after uploading to pypi actually

Reply:
Also, can you include instructions saying to solicit feedback on the release notes before creating the github release